tests/converter/dialogflow_sample_project/intents/account.spending.check - context_ spending - date.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Are the spaces in the file name intentional?

Reply:
Yes, since this is the file name that is generated from DF